csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -185,15 +184,30 @@ private void GenerateSourceFilesFromWebViews()
 
         public DependencyManager(string srcDir) : this(srcDir, DependencyOptions.Default, new ConsoleLogger(Verbosity.Info)) { }
 
-        private IEnumerable<string> GetFiles(string pattern, bool recurseSubdirectories = true) =>
+        private IEnumerable<FileInfo> GetFiles(string pattern, bool recurseSubdirectories = true) =>
```

Comment:
This method could probably be simplified to `GetAllFiles()` if `GetFiles("nuget.config", recurseSubdirectories: false)` is refactored.